src/client/app/containers/MapChartContainer.ts

```diff
@@ -339,14 +349,10 @@ function mapStateToProps(state: State) {
 	 *               layout={layout}
 	 *               onClick={({points, event}) => console.log(points, event)}>
 	 */
-	const props: any = {
+	const props = {
 		data,
-		layout,
-		config: {
-			locales: Locales // makes locales available for use
-		}
-	};
-	props.config.locale = state.options.selectedLanguage;
+		layout
```

Comment:
I think removing the locales is why plotly is no longer using the chose language to label the axis items.

Reply:
Indeed, Corrected the issue.